presto-main/src/main/resources/webapp/query.html

```diff
@@ -170,7 +170,9 @@
     }
 
     if (query.failureInfo) {
-        d3.select('#failureMessage').text(query.failureInfo.type + ": " + query.failureInfo.message);
+        d3.select('#failureMessage').html(
+            htmlEscape(query.failureInfo.type) + ": " + htmlEscape(query.failureInfo.message) + "<br>"
+            + htmlEscape(query.failureInfo.stack.join("\n")).replace(/\n/g,'<br>'));
```

Comment:
This needs to be rendered in the standard format so it can be copy/pasted into tools like IntelliJ. You can see the code for this in the JDK in `Throwable.printStackTrace()`.

Each stack element line is prefixed with a tab followed by "at ". Try doing this in HTML by putting the whole thing in a `<pre>` block. If that doesn't work, just use four spaces.

This also needs to handle chained exceptions via the `cause` field and suppressed exceptions via the `suppressed` list.

An easy way to see a printed stack trace for a given exception `e` is to use `e.printStackTrace(System.out);`.